### PR TITLE
Revert visual regression

### LINKED ISSUE #10375: Visual regression introduced in #10127
This is indeed visible when pulling the report from the commit (https://github.com/jupyterlab/jupyterlab/commit/d3915f659e76fedd0516da92925431e3e0b3c17b) that merged #10127: 

https://github.com/jupyterlab/jupyterlab/runs/2775919871

We can see the difference before and after:

https://user-images.githubusercontent.com/591645/121357917-7dbcb900-c932-11eb-9f2a-3d62593e4d57.mp4

_Originally posted by @jtpio in https://github.com/jupyterlab/jupyterlab/issues/10372#issuecomment-857669625_

### PR DESCRIPTION
**Pull request by fcollonval**
## References

Correct regression introduced in #10127
Fixes #10375

Add a new step in the ui test job to generate reference candidate in case the UI tests are failing. This will reduce the burden by not having to change the CI file twice to be able to update the reference screenshots.

## Code changes

N/A

## User-facing changes

Restore cell collapser

## Backwards-incompatible changes

N/A

### CODE REVIEW

**Comment by fcollonval** [comment]
UI tests should fail (see discussion https://github.com/jupyterlab/jupyterlab/pull/10372).

**Comment by jtpio** [comment]
> UI tests should fail (see discussion #10372).

Yes they do: https://github.com/jupyterlab/jupyterlab/pull/10376/checks?check_run_id=2785010036

image

File: .github/workflows/ui-tests.yml [hunk 1]
```diff
@@ -13,74 +13,91 @@ jobs:
...
+      - name: Run UI Tests
+        if: ${{ failure() }}
+        run: |
+          cd ui-tests
+          [ -d "jlab_root" ] && rm -rf jlab_root
+          mkdir jlab_root
+          jlpm run test:create-references
+          rm -rf jlab_root
+
+      - name: Upload UI Test new reference artifacts
```
Comment (jtpio) [inline .github/workflows/ui-tests.yml:97]: Nice, this will be really handy :+1:

**Comment by fcollonval** [comment]
We are finally green on this one.

**Comment by jtpio** [comment]
Looks good on the screenshot diff :+1: 

image

**Review by jtpio — APPROVED** [review]
Left a minor wording comment above, otherwise looks good.

Thanks!

**Comment by fcollonval** [comment]
Thanks @jtpio for correcting the wording.

I'll merge this one.

**Pull request merged**